doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt

```diff
@@ -355,8 +355,8 @@ PyWavelets==1.4.1
 PyYAML==6.0.1
 pyzmq==24.0.1
 querystring-parser==1.2.4
-ray @ file:///home/ray/ray-2.9.2-cp39-cp39-manylinux2014_x86_64.whl#sha256=7e073e8f1c465064c40c9318376b5bd4568589f2c7ae4e3bc5f86a4e7b0b2fe0
-ray-cpp==2.9.2
+ray @ file:///home/ray/ray-2.9.3-cp39-cp39-manylinux2014_x86_64.whl#sha256=7e073e8f1c465064c40c9318376b5bd4568589f2c7ae4e3bc5f86a4e7b0b2fe0
+ray-cpp==2.9.3
```

Comment:
I think those pip freeze files are added when we update the docker dependencies in  this [release step](https://www.notion.so/anyscale-hq/How-to-release-Ray-go-release-ray-28e4a2347577471288bbd7d64001fa88?pvs=4#9cc39bb28f5742778b62dd1381601e03) For now let's take it out. Same for the other one

Reply:
removed, thanks!